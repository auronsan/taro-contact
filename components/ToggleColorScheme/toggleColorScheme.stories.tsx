import React from 'react';
import { ToggleColorScheme } from './index';

export default {
  title: 'components/ToggleColorScheme',
  component: ToggleColorScheme,
};

export const Default = () => <ToggleColorScheme />;
