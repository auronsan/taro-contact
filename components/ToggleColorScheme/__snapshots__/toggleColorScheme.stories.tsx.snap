// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/ToggleColorScheme Default smoke-test 1`] = `
<main class="roboto-normal">
  <div class="actionIcon_root__jjWKK"
       role="button"
  >
    <div class="group_root__JMRrN group_gap-sm__VCWmB">
      <svg xmlns="http://www.w3.org/2000/svg"
           width="24"
           height="24"
           viewbox="0 0 24 24"
           fill="none"
           stroke="currentColor"
           stroke-width="2"
           stroke-linecap="round"
           stroke-linejoin="round"
           class="tabler-icon tabler-icon-moon "
      >
        <path d="M12 3c.132 0 .263 0 .393 0a7.5 7.5 0 0 0 7.92 12.446a9 9 0 1 1 -8.313 -12.454z">
        </path>
      </svg>
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
